x-pack/plugins/cases/server/client/utils.ts

```diff
@@ -131,6 +138,13 @@ const decodeExternalReferenceAttachment = (attachment: CommentRequestExternalRef
       fold(throwErrors(badRequest), identity)
     );
   }
+
+  const metadata = attachment.externalReferenceMetadata;
```

Comment:
nit: variable declaration could be moved inside the `if`. It isn't used outside.